### PR TITLE
sample fix for o(n!) of combinatorial complex

### PR DESCRIPTION
**Pull request by bahramiramazan**
@mhajij  Combinatorial_complex has  an attribute named ' _aux_complex ' , that is an object of simplicial_complex. Adding a cell into the  combinatorial_complex , basically involves creating a simplex , and then  passing it to add_cell method of '_aux_complex'. I had to change all the three in order to add a cell of order 17 or above .However  for combinatorial complex  the coadjacency does not work after modification, and for sure it have consequences for simplicial_complex.

### CODE REVIEW

File: toponetx/classes/simplex.py [hunk 1]
```diff
@@ -47,8 +47,8 @@ def __init__(self, elements, name=None, construct_tree=True, **attr):
         if len(self.nodes) != len(elements):
             raise ValueError("A simplex cannot contain duplicate nodes.")
 
-        if construct_tree:
-            self._faces = self.construct_simplex_tree(elements)
+        # if construct_tree:
```
Comment (ninamiolane) [inline toponetx/classes/simplex.py:50]: Remove commented code.

**Comment by mhajij** [comment]
@bahramiramazan  since the problem you raised above is in the combinatorialcomplex class, then why are you modifying the simplicialcomplex class?

**Comment by ninamiolane** [comment]
@bahramiramazan all checks have failed (see above): they would need to pass before we can merge. Thanks!

**Comment by bahramiramazan** [comment]
> @bahramiramazan all checks have failed (see above): they would need to pass before we can merge. Thanks!

well noted ! thanks

**Pull request closed without merging**